crates/ruma-api-macros/src/api/request/outgoing.rs

```diff
@@ -34,9 +35,25 @@ impl Request {
                     &format_string[start_of_segment + 1..end_of_segment],
                     Span::call_site(),
                 );
+                let path_field_string = self
```

Comment:
Can you revert the change in this file? It's a bug for a lint like `str_to_string` to fire on macro-generated code and we definitely shouldn't make our codegen more complex just to have the code that small bit nicer.

Reply:
To be clear: Let's just leave out `str_to_string` from the alias if it's buggy.